rplugin/python3/molten/outputbuffer.py

```diff
@@ -218,11 +218,38 @@ def show_virtual_output(self, anchor: Position) -> None:
         )
         self.canvas.present()
 
+    def calculate_offset(self, anchor: Position) -> int:
+        current_pos = anchor
+        offset = 0
+        while current_pos.lineno > 0:
+            current_line = self.nvim.funcs.nvim_buf_get_lines(
+                current_pos.bufno,
+                current_pos.lineno,
+                current_pos.lineno + 1,
+                False,
+            )[0]
+
+            if current_line != "" and not current_line.startswith(self.options.comment_string):
```

Comment:
I'm having problems when the comment string isn't set bc the default value is `""` and all strings start with `""`.  Solution is probably to have it default to None.

But I think a better solution is to make it a list, and have it default to empty. That way if people who're using this plugin for more than one notebook language, they wouldn't have to change the configuration around.

<img width="918" alt="image" src="https://github.com/benlubas/molten-nvim/assets/56943754/f2b0a2b8-b5ba-4d49-85e4-a55fa2451574">

Reply:
Wow. Thanks, can't believe I made that mistake. Of course every line starts with an empty string. I forgot to change my settings and remove that config option. I changed it and renamed the option to more accurately reflect what it does, it wasn't super clear what it was used for otherwise.